R/fa.R

```diff
@@ -55,7 +65,9 @@ fa <- function(name,
                height = NULL,
                width = NULL,
                margin_right = NULL,
-               position = NULL) {
+               position = NULL,
+               title = NULL,
+               a11y = c("decorative", "semantic", "none")) {
```

Comment:
Maaaaybe: `c("d", "s", "n")`

So users can do:

```R
fa("R-project", a11y = "s")
```

instead of

```R
fa("R-project", a11y = "semantic")
```